packages/botonic-react/src/index.d.ts

```diff
@@ -198,11 +201,37 @@ export class WebchatApp {
   updateWebchatSettings(settings: WebchatSettingsProps): void
 }
 
+export interface WebChatContextProps {
+  sendText: (text: string) => string
+  sendAttachment: (attachment: File) => string
+  sendPayload: (payload: string) => string
+  sendInput: (input: core.Input) => string
+  openWebview: (webviewComponent: Webview) => string
+  addMessage: (message: WebchatMessage) => string
+  updateMessage: (message: WebchatMessage) => string
+  updateReplies: (replies: boolean) => string
+  updateLatestInput: (input: core.Input) => string
+  closeWebview: () => string
+  toggleWebchat: () => string
+  getThemeProperty: (
+    property: string,
+    defaultValue?: string
+  ) => string | undefined
+  resolveCase: () => string
+  theme: ThemeProps
+  webchatState: WebchatState
+  updateWebchatDevSettings: (settings: WebchatSettingsProps) => void
+  updateUser: (user: core.SessionUser) => string
```

Comment:
here for example, i would expect `updateUser` to return `void` like the previous one